client/client.lua

```diff
@@ -61,33 +59,64 @@ RegisterNUICallback("PurchaseCart", function(data, cb)
     cb(1)
 end)
 
+local function onEnter(self)
+    if not IsPedInAnyVehicle(cache.ped, false) then return end
```

Comment:
Instead of doing 

`if not IsPedInAnyVehicle(cache.ped, false) then return end`

do

`if not cache.vehicle then return end`